src/components/UserRegister.tsx

```diff
@@ -0,0 +1,278 @@
+import React, { useEffect, useState } from "react";
+import { useForm } from "react-hook-form";
+import { makeStyles, createStyles, Theme } from "@material-ui/core/styles";
+import LoadingOverlay from "react-loading-overlay-ts";
+import Grid from "@material-ui/core/Grid";
+import TextField from "@material-ui/core/TextField";
+import Australia from "/home/ishfaq/training-app/src/flags/Australia.png";
+import america from "/home/ishfaq/training-app/src/flags/america.png";
+import FlagSelect from "./FlagSelect";
+import Logo from "./Logo.png";
+import { IFormInput } from "./interfaces";
+
+import { useHistory } from "react-router-dom";
+
+const flags = {
```

Comment:
you could reuse this one

Reply:
I think we need to create this flags array at some common point so we could reuse it at multiple places.